stories/emailTemplates/NotificationDigestEmail.stories.tsx

```diff
@@ -0,0 +1,194 @@
+import { Meta, StoryObj } from "@storybook/react"
+import digestEmail from "functions/src/email/digestEmail.handlebars"
+import { EmailTemplateRenderer } from "./EmailTemplateRenderer"
+import {
+  BillDigest,
+  BillResult,
+  NotificationEmailDigest,
+  UserDigest
+} from "functions/src/notifications/emailTypes"
+import { Frequency } from "components/auth"
+
+const meta: Meta = {
+  title: "Email Templates/Notifications Digest",
+  component: EmailTemplateRenderer,
+  decorators: [
+    Story => (
+      // Use a narrow window to represent an email client
+      // TODO: May need to update to match styling of a real email client
```

Comment:
This is likely never going to match the full styling of the wide array of real email clients - but given we're going with a mobile first design anyway, this is likely worthwhile.

You can always remove this decorator to test the desktop view (or there may be other Storybook configs for it - I'm not super well-versed in those options).